src/styles/components/collapsible-error-message.less

```diff
@@ -0,0 +1,89 @@
+
+//
+// The CollapsibleErrorMessage is a component used on top
+// of forms or on other places to display a main error
+// message and a toggled drop-down with the error details
+//
```

Comment:
Since all of our LESS is being refactored in an upcoming release, these comments don't need to change. But just a forewarning, in the future we are planning to adhere to three tiers of comment blocks. Stay tuned for a CSS style guide :)